### PR TITLE
Update usaco-104.mdx

### PR DESCRIPTION
**Pull request by saa938**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [✔] I have tested my code.
- [✔] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [✔] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
> * I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions)

read them again

File: solutions/usaco-104.mdx [hunk 1]
```diff
@@ -87,5 +87,26 @@ class haybaleStacking {
...
+    start, end = map(int, input().split())
+    differences[start-1] += 1 # add one to the start
+    differences[end] -= 1 # subtract one from the next number outside of start-end
+pref = [0]*n # prefix sums array
+s = 0	
+for i in range(n): # uses differences and total sum to get where all the haybales are
+    pref[i] = differences[i] + s 
+    s += differences[i]
+pref.sort() 
+print(pref[n//2])
```
Comment (ChuangSheep) [inline solutions/usaco-104.mdx:107]: same

File: solutions/usaco-104.mdx [hunk 2]
```diff
@@ -87,5 +87,26 @@ class haybaleStacking {
...
+```
+	
+</PythonSection>
```
Comment (ChuangSheep) [inline solutions/usaco-104.mdx:110]:
```suggestion
</PySection>
```

**Review by ChuangSheep — CHANGES_REQUESTED** [review]
and use **tab** to indent

**Pull request closed without merging**